src/main/resources/public/js/pages/app-install.js

```diff
@@ -130,144 +135,167 @@ export default class AppInstall extends React.Component {
 
         return (
             <div className={'app-install-wrapper'}>
-                <UpdateTitle title={app.name}/>
+                <UpdateTitle title={app.name} />
                 <div className={'flex-row header-app-install'}>
                     <div className={'information-app flex-row'}>
-                        <img alt={'app icon'} src={app.icon}/>
+                        <img alt={'app icon'} src={app.icon} />
                         <div className={'information-app-details'}>
-                            <p><strong>{app.name}</strong></p>
-                            <p>{app.provider}</p>
-                            <p>{app.description}
+                            <div className="information-app-name">
+                                <strong>{app.name}</strong>
+                            </div>
+                            <div className="information-app-provider">{app.provider}</div>
+                            <div className="information-app-description">
+                                {app.description}
                                 &nbsp;
-                                {
-                                    appDetails.longdescription === app.description ?
-                                        null :
-                                        <i className="fas fa-external-link-alt go-to-long-description"
-                                           onClick={this.scrollToLongDescription}/>
-                                }
-                            </p>
+                                {appDetails.longdescription === app.description ? null : (
+                                    <i
+                                        // className="fas fa-external-link-alt go-to-long-description"
```

Comment:
Does this code is supposed to be commented ? Remove it if it's not useful

Reply:
Removed!